classes/Domain/Services/Authentication.php

```diff
@@ -50,7 +50,7 @@ public function userId(): int;
      *
      * @return bool
      */
-    public function check(): bool;
+    public function isAuthenticated(): bool;
```

Comment:
Also see #835.

Reply:
@BackEndTea @chartjes @mdwheele 

Since we have

* [`authenticate()`](https://github.com/opencfp/opencfp/blob/d872d646e9b561e9cb8ed41fc3e1a4abdbaddfeb/classes/Domain/Services/Authentication.php#L20-L28)
* [`check()`](https://github.com/opencfp/opencfp/blob/d872d646e9b561e9cb8ed41fc3e1a4abdbaddfeb/classes/Domain/Services/Authentication.php#L48-L53)
* [`guest()`](https://github.com/opencfp/opencfp/blob/d872d646e9b561e9cb8ed41fc3e1a4abdbaddfeb/classes/Domain/Services/Authentication.php#L55-L60), also see #835 
* [`logout()`](https://github.com/opencfp/opencfp/blob/d872d646e9b561e9cb8ed41fc3e1a4abdbaddfeb/classes/Domain/Services/Authentication.php#L62-L65)

what do you think about aligning them altogether?

### Example 1

* `authenticate()`
* `isAuthenticated()`
* `clear()`

### Example 2

* `signIn()`
* `isSignedIn()`
* `signOut()`

### Example 3

* `logIn()`
* `isLoggedIn()`
* `logOut()`

?